apps/condo/domains/banking/components/BankAccountInfo.tsx

```diff
@@ -47,19 +47,30 @@ export const BankingInfo: React.FC<IBankAccountInfo> = ({ bankAccount }) => {
 
     return (
         <RequisitesContainer style={containerStyle}>
-            <Tag
-                bgColor={ colors.gray[7]}
-                textColor={colors.white}
-            >
-                {accrualsAndPayments}
-            </Tag>
-            <Row gutter={GUTTER}>
-                <Typography.Title level={3}>{get(bankAccount, 'organization.name', '-')}</Typography.Title>
+            <Row gutter={[0, 22]}>
```

Comment:
Why 22 instead of 20? Is it according to the design?

Reply:
I don’t know, I counted pixel by pixel in Figma)